packages/dashboard/src/utils/notifications/analytics.tsx

```diff
@@ -0,0 +1,71 @@
+import { Stack, Group, Button, Text, Anchor } from "@mantine/core";
+import type { NotificationProps } from "@mantine/notifications";
+
+type NotificationPropsWithId = NotificationProps & { id: string };
+
+export const analyticsNotificationId = "analytics";
+
+const baseNotification = {
+  autoClose: false,
+  disallowClose: true,
+  id: analyticsNotificationId
+};
+
+export const analyticsNotifications: Record<
+  "ask" | "ask-again" | "thank",
+  (enable?: () => void, disable?: () => void) => NotificationPropsWithId
+> = {
+  "ask": (enable, disable) => ({
+    ...baseNotification,
+    title: "Help improve Truffle",
+    message: (
+      <Stack>
+        <Text>
+          You can help us improve Truffle by allowing us to track how you use
+          our tools. Would you like to share anonymous telemetry? View the
+          Truffle Analytics policy&nbsp;
```

Comment:
Truffle Telemetry policy?

Reply:
I think we're collecting telemetry and it's governed by our analytics policy. Right?

Reply:
thank you for clarifying for me